test/bin/mdx-test/expect/env-and-prelude/test-case.opts

```diff
@@ -1 +1 @@
---prelude default.ml --prelude a:a.ml --prelude b:b.ml
+--prelude all.ml --prelude " :default.ml" --prelude a:a.ml --prelude b:b.ml
```

Comment:
I realize now that this tests is a bit hard to understand. How about we change it so that each prelude defines a single value that has the same name as the prelude file and we try to evaluate each variables in each blocks so that it's clear which one is available in which env.

To clarify I'm suggesting using this block:
````markdown
```ocaml
# all;;
- : string = "all"
# default;;
# a;;
# b;;
```
````
with all four envs.

I think I wrote this test as a showcase of a bug and looking at it now I'm a bit lost. I'm not sure the `x` part is relevant to what we're testing.

What do you think?

Reply:
I'm all for it, I made the change